File: 백준/Bronze/11948. 과목선택/과목선택.js
const fs = require('fs');
const [a, b, c, d, e, f] = fs.readFileSync(0, 'utf8').trim().split('\n').map(Number);
const topA = [a, b, c, d].sort((a, b) => b - a).slice(0, 3).reduce((sum, n) => sum + n, 0);
const topB = Math.max(e, f);

console.log(topA + topB)

File: 백준/Bronze/11948. 과목선택/과목선택.py
import sys

input = sys.stdin.read().split()
a, b, c, d, e, f = map(int, input)

top_a = sorted([a, b, c, d], reverse=True)[:3]
top_b = max(e, f)

print(sum(top_a) + top_b)